src/components/manage/BlockChooser/BlockChooserButton.jsx

```diff
@@ -75,7 +74,7 @@ const BlockChooserButton = (props) => {
 
   return (
     <>
-      {!disableNewBlocks && !blockHasValue(data) && (
+      {!disableNewBlocks && (
```

Comment:
Note to self: make sure disableNewBlocks still works

Reply:
@davisagli I'm not sure how to test this in vanila. Maybe @avoinea knows?

Reply:
It's available as an option if I enable editable blocks for a content type and edit the layout. It's currently not working, but I think it should be easy to fix.

Reply:
Actually it's working fine here (add button does not show for the block with this setting turned on) but we are missing checks for disableNewBlocks if the user creates a new block using Enter or clicking below all blocks.